CRM/Contribute/Form/AdditionalPayment.php

```diff
@@ -363,7 +363,6 @@ public function processCreditCard() {
     $session = CRM_Core_Session::singleton();
 
     $now = date('YmdHis');
```

Comment:
@JKingsnorth Looks like `$now` is never used either?

Reply:
$now is used further down, line 483

Reply:
Scratch that! Looking at old version of the repo!